AnswerKingAPI.yml

```diff
@@ -266,32 +266,32 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/AnswerKing.RequestModels.OrderDto'
+              $ref: '#/components/schemas/Order.Request'
           text/json:
             schema:
-              $ref: '#/components/schemas/AnswerKing.RequestModels.OrderDto'
+              $ref: '#/components/schemas/Order.Request'
           application/*+json:
             schema:
-              $ref: '#/components/schemas/AnswerKing.RequestModels.OrderDto'
+              $ref: '#/components/schemas/Order.Request'
       responses:
         '200':
           description: When the order has been updated.
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/AnswerKing.Orders.Order'
+                $ref: '#/components/schemas/Orders.Order.Response'
```

Comment:
I think plain Order.Response should be OK.

Reply:
its kept this way to be consistant with the other models within Orders i.e. `Orders.Product.Product` and Orders.Category.Product` since they are all in the domain of `Orders`